valinta-tulos-henkiloviite-synchronizer/src/main/scala/fi/vm/sade/valintatulosservice/henkiloviiteClient.scala

```diff
@@ -2,54 +2,70 @@ package fi.vm.sade.valintatulosservice
 
 import java.util.concurrent.TimeUnit
 
-import fi.vm.sade.utils.cas.{CasAuthenticatingClient, CasClient, CasParams}
-import org.http4s.Status.ResponseClass.Successful
-import org.http4s._
-import org.http4s.client.Client
-import org.http4s.headers.`Content-Type`
-import org.http4s.json4s.native.jsonOf
+import fi.vm.sade.javautils.nio.cas.{CasClient, CasClientBuilder}
+import fi.vm.sade.security.ScalaCasConfig
+import org.asynchttpclient.{RequestBuilder, Response}
+import org.json4s.native.JsonMethods.parse
 import org.json4s.DefaultReaders.{StringReader, arrayReader}
 import org.json4s.JsonAST.JValue
-import org.json4s.Reader
+import org.json4s.{Reader, Formats, DefaultFormats}
+import org.http4s.json4s.native.jsonOf  // TODO get rid of this
```

Comment:
Tässä todot

Reply:
Aaaaa ok, en tajunnut että "todo" viittasi koodin sisältöön, ajattelin että sulla on jossain joku lista jossa on TODO-kohtia mitä pitää tehdä ennen kuin tää menee läpi :D 

Anyway, olin tarkoittanut noi tulevaisuuteen mut ehkä toteutan ne sitten tässä.  Se nimittäin taitaa olla kuollutta koodia missä noita käytetään.

Reply:
Jes, poistin kuolleen koodin testeineen ja myös nuo  importit.